### PR TITLE
build: update vscode settings to support clang-format

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+  "files.autoSave": "onFocusChange",
+  "editor.formatOnSave": true,
+  // Please install http://clang.llvm.org/docs/ClangFormat.html in VSCode to take advantage of clang-format
+  "clang-format.executable": "${workspaceRoot}/node_modules/.bin/clang-format",
+  "breadcrumbs.enabled": true,
```
Comment (alexeagle) [inline .vscode/settings.json]: this seems like your personal preference. what's wrong with me doing angular development with no breadcrumbs?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,9 @@
 {
+  "files.autoSave": "onFocusChange",
```
Comment (alexeagle) [inline .vscode/settings.json]: looks like a personal preference

File: .vscode/settings.json (hunk 1 above)
Comment (mhevery) [reply .vscode/settings.json]: removed

File: .vscode/settings.json (hunk 2 above)
Comment (mhevery) [reply .vscode/settings.json]: removed

**Review by alexeagle — APPROVED** [review]

**Comment by mhevery** [comment]
The failure is a flake. This change does not touch any code (only VSCode configuration)

**Pull request closed without merging**